### DISCUSSION TITLE
Naming things: CPM-ified org repos

### DISCUSSION THREAD

**Discussion by iboB** [cpm-cmake/CPM.cmake#196] [General]
What should be the convention for naming repos which are CPM-ified builds for existing projects which either don't use CMake or do, but it can't be used with CPM?

For example, boost:

* just **boost**
    * assuming that the project url `https://github.com/cpm-cmake/boost` will serve as a proper namespace
    * probably will be unpleasant to fork as the default name of the fork will be "boost" which may clash with existing repos with the same name
* **boost-cpm**
   * may also clash with a sub-project of the project which is unrelated, but has the name cpm. The convention `<super>-<sub>` is popular (say `poco-websocket`)
* **cpm-boost**
   * may be unpleasant if there is ever have a CMake package as by this convention it will be named `cpm-cmake` 😄 
* **boost.cpm**
   * specifically for boost this is probably not a good idea, as the boost lib naming convention is `boost.<lib>`, say `boost.asio`
* Something else?

**Comment by TheLartians** [comment]
I'd say it depends. 
- If we're adding CMake to a project that previously didn't have support it previously, I like the `-cmake` most (and would consistent with previous efforts, e.g. `Orphis/boost-cmake`). In general I think this would also be useful for non-CPM.cmake users, so, for better discoverability, the naming would indicate that it's not CPM.cmake exclusive.
- Otherwise, when making am existing project compatible with CPM.cmake, I'd vote for the `-cpm` suffix.
